.github/workflows/test-back-end.yml

```diff
@@ -0,0 +1,36 @@
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - "*"
+
+name: Test Back End
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Make shared env
+              uses: ./.github/actions/make-shared-env
+              with:
+                install-packages: 'false'
+
+            - name: Run deps
+              run: scripts/run-dependencies.sh
+
+            - name: Prep back end tests
+              run: |
+                mkdir -p src/app/static/public
+                mkdir -p src/app/static/public/css
+                yes ".class {height: 500px;}" | head -n 200 >> src/app/static/public/css/app.css
```

Comment:
Basically here I am making a dummy build of the app, turns out that the backend tests only need the freemarker template files and the css file, this css file needs to be big enough for gzip to activate, so I am creating a very bad css file that has `.class {height: 500px;}` written 200 times

Reply:
Can we document this in the yml? Think you can do this at the top of a run block using
```
: # Create dummy build of app...
```

Reply:
done